src/OpenConext/ProfileBundle/Consent/BlacklistFilter.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenConext\ProfileBundle\Consent;
+
+use Surfnet\SamlBundle\SAML2\Attribute\Attribute;
+use Surfnet\SamlBundle\SAML2\Attribute\Filter\AttributeFilter as AttributeFilterInterface;
+
+final class BlacklistFilter implements AttributeFilterInterface
+{
+    /**
+     * @var array
+     */
+    const BLACKLIST = [
+        'urn:oid:2.5.4.42',
```

Comment:
You can be more specific by writing `@var string[]`. :wink:

Not sure if this qualifies as a constant, but since this list is unchanged since 2011, I think I'm ok with this. :stuck_out_tongue: